storage/scylla_api.h

```diff
@@ -0,0 +1,28 @@
+#ifndef IOTA_SCYLLA_API_H_
+#define IOTA_SCYLLA_API_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "accelerator/common_core.h"
+
+/**
+ * Initialize logger
+ */
+void scylla_api_logger_init();
+
+/**
+ * Release logger
+ *
+ * @return
+ * - zero on success
+ * - EXIT_FAILURE on error
+ */
+int scylla_api_logger_release();
+
+int scylla_test(char* hosts);
```

Comment:
Test file should be in `tests/` directory.

Reply:
I added tests/test_scylla.c for testing.